.circleci/config.yml

```diff
@@ -176,13 +176,28 @@ commands:
             .\venv\Scripts\activate
             pip install --upgrade --force-reinstall --no-deps .
           shell: powershell.exe
+
+      - run:
+          name: install gymansium[mujoco]
+          command: |
+            .\venv\Scripts\activate
+            pip install gymnasium[mujoco]
+          shell: powershell.exe
+
       - run:
           name: print installed packages
           command: |
             .\venv\Scripts\activate
             pip freeze --all
           shell: powershell.exe
 
+      - run:
```

Comment:
Well done tracking down this issue and the workaround! (I had no idea about this Windows limitation.)